conda/recipes/versions.yaml

```diff
@@ -1,7 +1,7 @@
 # Versions for `rapids-xgboost` meta-pkg
 xgboost_version:
   # Minor version is appended in meta.yaml
-  - '=1.7.5dev.rapidsai'
+  - '=1.7.4'
```

Comment:
For context we were building dev versions before (and applying patches). Those patches were largely upstreamed (though we still have a couple patches)

However we now build a stable version of XGBoost (instead of from a commit as before). Hence 1.7.4 is reasonable

Reply:
Thanks!